devices/wholeBodyDynamics/KnownExternalWrench.h

```diff
@@ -0,0 +1,95 @@
+#ifndef KNOWNEXTERNALWRENCH_H
+#define KNOWNEXTERNALWRENCH_H
+
+#include <array>
+#include <vector>
+
+#include <iDynTree/Estimation/ExtWrenchesAndJointTorquesEstimator.h>
+//#include <iDynTree/Core/Position.h>
+//#include <iDynTree/Core/Direction.h>
+
+namespace wholeBodyDynamics
+{
+
+/**
+ * Class for processing the raw measure of
+ * a Six Axis Force Torque sensor.
+ *
+ * In particular this class process the raw measurement of a
+ * Six Axis FT sensor using an affine function:
+ *
+ * ftNew = secondaryCalibrationMatrix*ftOld - offset
+ *
+ * Where offset is the offset of the F/T sensor (tipically computed
+ * online at given instants where only one external force is known
+ * to be acting on the robot, while secondaryCalibrationMatrix is a
+ * secondary calibration matrix, that is tipically set to the identity.
+ *
+ */
+class KnownExternalWrench
+{
+private:
+    std::string frameName;
+    std::string type;
+    std::vector<double> position;
+    std::vector<double> direction;
```

Comment:
Why the choice of using `std::vector` instead of `iDynTree::Position` and `iDynTree::Direction`?

Reply:
In case, you choose to avoid `iDynTree` objects, then I would suggest using `std::array` instead of `std::vector`. In this way, we make a decision already about the size of the object already during compile-time.

Reply:
at the end I settled with `iDynTree` objects (in https://github.com/robotology/whole-body-estimators/pull/48/commits/edcd509dad868124ec98e627ecd260af425c8ca1)